src/wallet/test/db_tests.cpp

```diff
@@ -228,6 +228,48 @@ BOOST_AUTO_TEST_CASE(db_availability_after_write_error)
     }
 }
 
+BOOST_AUTO_TEST_CASE(concurrent_txn_dont_interfere)
+{
+    std::string key = "key";
+    std::string value = "value";
+    std::string value2 = "value_2";
+
+    DatabaseOptions options;
+    DatabaseStatus status;
+    bilingual_str error;
+    const auto& database = MakeSQLiteDatabase(m_path_root / "sqlite", options, status, error);
```

Comment:
This will fail to compile without sqlite support. Need to scope the test under the `USE_SQLITE` compile flag. 
```diff
diff --git a/src/wallet/test/db_tests.cpp b/src/wallet/test/db_tests.cpp
--- a/src/wallet/test/db_tests.cpp	(revision f1b5b259f6d4759e7fc514b82b4096f7669e8325)
+++ b/src/wallet/test/db_tests.cpp	(date 1704374489489)
@@ -228,6 +228,7 @@
     }
 }
 
+#ifdef USE_SQLITE
 BOOST_AUTO_TEST_CASE(concurrent_txn_dont_interfere)
 {
     std::string key = "key";
@@ -269,6 +270,7 @@
     BOOST_CHECK(handler2->Read(key, read_value));
     BOOST_CHECK_EQUAL(read_value, value2);
 }
+#endif // USE_SQLITE
 
 
 BOOST_AUTO_TEST_SUITE_END()
```

Reply:
Done